docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+
+Bear in mind that when a pull request is made to fix this issue, the fixer and the PR's tester will both be relying on the information you provide here to make their lives easier.
+
+### 7. Copy across to AC tracker with link back to CC
+So now you have established this is a valid bug, you can move it across to the AzerothCore issue tracker. This is done by creating a new blank issue here: https://github.com/azerothcore/azerothcore-wotlk/issues/new?assignees=&labels=&template=
+You can then copy the data from the original CC ticket. You can do this by clicking the three horizontal dots in the top right corner of the report's text box and selecting Edit. You can then copy the ticket text and paste that information into the AC ticket, and then add your own comments and findings to generate the final ticket. 
+
+Title it appropriately - something clear and consistent is best. Then add the link to the original issue reported to ChromieCraft ("Originally reported LINK-TO-CHROMIECRAFT-ISSUE"). GitHub will automatically link the two reports. 
+
+You should also check to make sure the CC report has a server version hash included. (The version hash is a little string of random numbers and letters, like `3d4befd`.) If it doesn't the AC tracker will complain. If it is missing for some reason, the current ChromieCraft version info can be found [here](https://github.com/chromiecraft/azerothcore-wotlk), or if you have replicated the issue on your own test server, you can use your own hash. This can be found by typing `server info` at the `AC>` command prompt of your running AC world server.
+
+### 8. Add tags to AC and CC issues
+Now you can add relevant tags to both the new AC issue you have created, and the CC issue you have been working from. Tag appropriately - 'Generic' is last resort, for issues that affect a broad range of levels. Otherwise, tag an issue by the level of the zone it occurs in, or the level of the quest or NPC it affects. If you're not sure of a zone's level, you can find a Wrath-era list of zones by level [here](https://wowpedia.fandom.com/wiki/Zones_by_level_(original)).
+
+For the AC issue, if you are copying an issue from CC then you can add the 'Confirmed' tag. Also feel free to add other relevant tags, such as 'Quest', 'Loot', 'DB', 'PVP', or one of of the class-specific tags. 
+
+For the CC issue, you can tag it with 'Confirmed', 'Linked to AC', and a tag indicating level range or Generic if that doesn't apply. Again, if there are other relevant tags such as 'Class Issue', feel free to add them.
+
+If you have the right permissions, also add the new AC issue to whichever project is relevant, usually the one matching the issue's level range.
+
+##### Priority Tags
```

Comment:
This can be 3.

Reply:
3 would put it on the same level as its superheading, but I've changed it to 4 which fits.